lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
pipelines are only invoked for matched routes, which ueberauth has none. Should the docs show plugging in the endpoint instead, or is the plug supposed to carry on downstream?